lib/msal-common/src/authority/RegionDiscovery.ts

```diff
@@ -38,19 +39,27 @@ export class RegionDiscovery {
                 if (response.status === ResponseCodes.httpBadRequest) {
                     const latestIMDSVersion = await this.getCurrentVersion();
                     if (!latestIMDSVersion) {
+                        regionDiscoveryMetadata.region_source = RegionDiscoverySources.FAILED_AUTO_DETECTION;
                         return null;
                     }
 
                     const response = await this.getRegionFromIMDS(latestIMDSVersion);
                     if (response.status === ResponseCodes.httpSuccess) {
                         autodetectedRegionName = response.body;
                     }
-                } 
+                }
+                
+                if (autodetectedRegionName) regionDiscoveryMetadata.region_source = RegionDiscoverySources.IMDS;
```

Comment:
Braces? I do not understand this statement. Is it supposed to be:
```suggestion
                if (autodetectedRegionName) {
                    regionDiscoveryMetadata.region_source = RegionDiscoverySources.IMDS;
                }
```
Same with the below. The whole block isn't very readable.